nixos/modules/services/web-apps/wordpress.nix

```diff
@@ -78,6 +79,9 @@ let
   siteOpts = { lib, name, ... }:
     {
       options = {
+
+        enable = mkEnableOption (lib.mdDoc "Wordpress web application.");
```

Comment:
intentional?

Reply:
Yep I want to use this for the most simple setup `services.wordpress.sites."localhost".enabled = true;`

Reply:
> Yep I want to use this for the most simple setup `services.wordpress.sites."localhost".enabled = true;`

Isn't `services.wordpress.sites."localhost" = {}` also possible (can't check)?

Reply:
Trailing period is redundant. `mkEnableOption` already adds one.

Reply:
Also `mkEnableOption` defaults to `false`.

Reply:
So it is okay that I add this line? :)

Reply:
You said you want default true, right? A default of true makes sense to me here too.

Reply:
Can you please give an example where mkEnableOption is set default = true? Couldn't find something like this in Dokuwiki, Nextcloud or Onlyoffice module :O

Reply:
I think it doesn't exist because its a kinda weird use case. I think I would just remove that option and then you should still be able to create a website with
`services.wordpress.sites."localhost" = {}`

Reply:
`environment.etc.<name>.enable` and `services.logrotate.paths.<name>.enable` are 2 examples that come to mind - I know there are more.

This pattern makes plenty of sense when the idea is that NixOS wants to provide some default configuration that some users may wish to disable. In this case... I don't personally see the need for it, but I can understand if others would. :man_shrugging: